js/blocks/controls/range.js

```diff
@@ -9,16 +9,13 @@ const BlockLabRangeControl = ( props ) => {
 
 	return (
 		<RangeControl
-			beforeIcon={ field.beforeIcon }
-			afterIcon={ field.afterIcon }
```

Comment:
These properties of `field` don't exist

Reply:
Keeping up with Gutenberg development is hard!

Reply:
Haha, yeah. I don't think this caused any problem 😄